Extension/background.js

```diff
@@ -41,23 +43,43 @@ function snagHtmlLink(pageDetails) {
 // When the extension is installed or upgraded ...
 chrome.runtime.onInstalled.addListener(function () {
   // Replace all rules ...
-  chrome.declarativeContent.onPageChanged.removeRules(undefined, function () {
-    // With a new rule ...
-    chrome.declarativeContent.onPageChanged.addRules([
-      {
-        // That fires when a page's URL has the right host
-        conditions: [
-          new chrome.declarativeContent.PageStateMatcher({
-            pageUrl: {
-              urlMatches: 'jiraprod.agfahealthcare.com/browse/[A-Z][A-Z0-9]+-[0-9]+'
-            },
-          })
-        ],
-        // And shows the extension's page action.
-        actions: [new chrome.declarativeContent.ShowPageAction()]
-      }
-    ]);
-  });
+  if (chrome.declarativeContent) {

```

Comment:
Firefox doesn't have declarativeContent, but we still get in here. We'll handle this differently for Firefox, on line 68.